dotcom-rendering/src/web/components/FixedMediumSlowXIIMPU.tsx

```diff
@@ -361,9 +313,12 @@ export const FixedMediumSlowXIIMPU = ({
 		default: {
 			const topThree = trails.slice(0, 3);
 			const remainingCards = trails.slice(3, 9);
-			const lengthIsEven = remainingCards.length % 2 === 0;
+			const evenLength =
+				remainingCards.length % 2 === 0
+					? remainingCards.length
+					: remainingCards.length - 1;
```

Comment:
There’s a more concise way to do this, but I thought it might make it harder to follow.

```ts
const evenLength = remainingCards.length - (remainingCards.length % 2)
```

Reply:
Is this affected to the gap css or just an unrelated diff?

Reply:
Drive-by, moved to #6025